fem/pfespace.hpp

```diff
@@ -335,6 +338,7 @@ class ParFiniteElementSpace : public FiniteElementSpace
    HYPRE_Int GetMyTDofOffset() const;
 
    virtual const Operator *GetProlongationMatrix() const;
+   virtual const Operator *GetLocalProlongationOperator() const;
```

Comment:
As mentioned by Will above, we need some comments to distinguish/explain `GetProlongationMatrix()` and `GetLocalProlongationOperator()` here.